string SensorName
uint16 SetAddr
uint8[] ParamList